src/main/resources/assets/tropicraft/models/block/papaya_stage0.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
It should in theory be possible to datagen these and have them inherit from the cocoa model, overriding the `cocoa` and `particle` textures- might simplify a bit 🙂